prometheus-to-sd/translator/translator.go

```diff
@@ -391,6 +392,9 @@ func extractValueType(mType dto.MetricType, originalDescriptor *v3.MetricDescrip
 	if mType == dto.MetricType_HISTOGRAM {
 		return "DISTRIBUTION"
 	}
+	if mType == dto.MetricType_UNTYPED {
```

Comment:
You should update also setValue function above with value extraction from UNTYPED metric

Reply:
done.